modules/installation-extracting-infraid.adoc

```diff
@@ -19,8 +30,9 @@ it.
 
 .Procedure
 
-* To extract the infrastructure name from the Ignition config file metadata, run
-the following command:
+ifeval::["{context}" == "installing-aws-user-infra"]
+* To extract and view the infrastructure name from the Ignition config file
+metadata, run the following command:
```

Comment:
This part seems to be shared by both AWS and GCP.  Why condition it on `"{context}" == "installing-aws-user-infra"` instead of sharing?

Reply:
This was an attempt to avoid forcing a single-numbered step in the AWS version of the procedure, since it became a two-step procedure for GCP. Since this, however, I have come across other numbered, single-step procedures in modules; I would rather just share this step here because it's getting awkward.

Reply:
According to the [mod docs guide](https://redhat-documentation.github.io/modular-docs/#creating-procedure-modules), single-step procedures shouldn't be numbered.

> For single-step procedures, use an unnumbered bullet instead of a numbered list.